client/src/app/site/motions/services/change-recommendation-repository.service.ts

```diff
@@ -113,24 +112,24 @@ export class ChangeRecommendationRepositoryService extends BaseRepository<ViewCh
      *
      * @param {ViewChangeReco} change
      */
-    public setAccepted(change: ViewChangeReco): Observable<MotionChangeReco> {
+    public async setAccepted(change: ViewChangeReco): Promise<MotionChangeReco> {
         const changeReco = change.changeRecommendation;
         changeReco.patchValues({
             rejected: false
         });
-        return this.dataSend.updateModel(changeReco, HTTPMethod.PATCH) as Observable<MotionChangeReco>;
+        return await this.dataSend.partialUpdateModel<MotionChangeReco>(changeReco);
     }
 
     /**
      * Sets a change recommendation to rejected.
      *
      * @param {ViewChangeReco} change
      */
-    public setRejected(change: ViewChangeReco): Observable<MotionChangeReco> {
+    public async setRejected(change: ViewChangeReco): Promise<MotionChangeReco> {
```

Comment:
If we change our HTTP-Communication completely to Promise to wait for 👍 or :-1: - this should be just ```Promise<void>``` or `id`